test/tbb/test_allocators.cpp

```diff
@@ -14,6 +14,17 @@
     limitations under the License.
 */
 
+#include <tbb/version.h> // For __TBB_GLIBCXX_VERSION and __TBB_CPP20_PRESENT
+
+// Intel LLVM compiler triggers a deprecation warning in the implementation of std::allocator_traits::destroy
+// inside Standard Library while using STL PMR containers since std::polymorphic_allocator::destroy is deprecated since C++20
+#define TEST_LLVM_COMPILER_PMR_DESTROY_DEPRECATED_BROKEN __INTEL_LLVM_COMPILER == 20250000 && __TBB_GLIBCXX_VERSION == 110000 && __TBB_CPP20_PRESENT
```

Comment:
Should we be only testing for the major Intel LLVM compiler version instead of "20250000" (i.e., 2025.0.0)?

Reply:
I think better to have workarounds as picky as possible. This way we won't miss other issues about deprecated stuff that might turn out as real issues in the library's code.